company-profiles/iterative.md

```diff
@@ -0,0 +1,65 @@
+# Iterative
+
+## Company blurb
+
+AI teams face challenges that require new technologies. We build these technologies. Existing data warehouses and data lakes do not fit unstructured datasets like text, images, and videos.
+
+✅ Check out the [Website](https://dvc.org/) and [Docs](http://dvc.org/doc)
+
+✅ Check out our open-source projects in [GitHub](http://github.com/iterative)
+
+✅ Finally, take a look at our [Blog](http://dvc.org/blog) and [YouTube](https://www.youtube.com/channel/UC37rp97Go-xIX3aNFVHhXfQ)
+
+
+## Company size
+
+45-100
+
+## Remote status
+
+100% remote:
+
+> Iterative is a 100% remote team. We currently span 19 countries and 18 time zones
+
+We value great collaboration and communication skills, both among internal teams and in how we interact with our users. We take care to balance and be responsive to the needs of our open source community as well as our enterprise customers.
+
+### WHAT WE OFFER
```

Comment:
FYI, these H3 won't render as a sub heading due to how the site is built.

Reply:
Replaced with bold `*X*` for those 2